sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala

```diff
@@ -302,6 +348,7 @@ object SidechainNodeViewHolder /*extends ScorexLogging with ScorexEncoding*/ {
     case class ApplyFunctionOnNodeView[HIS, MS, VL, MP, A](f: java.util.function.Function[SidechainNodeView, A])
     case class ApplyBiFunctionOnNodeView[HIS, MS, VL, MP, T, A](f: java.util.function.BiFunction[SidechainNodeView, T, A], functionParameter: T)
     case class LocallyGeneratedSecret[S <: SidechainTypes#SCS](secret: S)
+    case class ApplyModifier(applied: Seq[SidechainBlock])
```

Comment:
let's make this one as a part of internal interface, because we don't want anyone outside to use this message. See `InternalReceivableMessages` in `CertificateSubmitter` object as an example.

Reply:
Cannot make it private because ApplyModifier is also used in com.horizen.mailbox.PrioritizedMailbox.

Reply:
Visibility was restricted to com.horizen package.